README.md

```diff
@@ -15,16 +15,20 @@ This tool is published as a multi-platform image on DockerHub as `mplatform/mque
 you can look up the `ubuntu:latest` image as follows:
```

Comment:
I like it, but this comment right before the example still says `ubuntu:latest` :)

Reply:
thank you, it should be ok right now 👍 @estesp